core/src/main/web/outputdisplay/bko-plot/bko-plot.js

```diff
@@ -544,17 +544,25 @@
           scope.jqcontainer.find("#legends").remove();
 
           scope.legendDone = true;
-          var legend = $("<table></table>").appendTo(scope.jqcontainer)
-            .attr("id", "legends")
-            .attr("class", "plot-legendcontainer")
+          var legendScrollableContainer = $("<div></div>").appendTo(scope.jqcontainer)
+            .attr("class", "plot-legendscrollablecontainer")
             .draggable({
               stop : function(event, ui) {
                 scope.legendPosition = {
                   "left" : ui.position.left,
                   "top" : ui.position.top
                 };
-              }
-            });
+              },
+              handle : "#legendDraggableContainer"
+            })
+            .css("max-height", scope.jqsvg.height() - scope.layout.bottomLayoutMargin);
+
+          var legendDraggableContainer = $("<div></div>").appendTo(legendScrollableContainer)
+            .attr("id", "legendDraggableContainer")
```

Comment:
It would be better not to reuse the ID but it looks like it works fine and before this PR it was already reusing the ID, so we don't have to fix it here and now.